module_utils/juniper_junos_common.py

```diff
@@ -497,12 +511,15 @@ class ModuleDocFragment(object):
 # Specify the logging spec.
 logging_spec = {
     'logfile': dict(type='path', required=False, default=None),
-    'logdir': dict(type='path', required=False, default=None)
+    'logdir': dict(type='path', required=False, default=None),
+    'level': dict(choices=[None, 'INFO', 'DEBUG'], required=False, default=None)
 }
 
 # The logdir and logfile options are mutually exclusive.
 logging_spec_mutually_exclusive = ['logfile', 'logdir']
 
+
+
```

Comment:
Why add extra unnecessary whitespace? Suggest you always run `pep8` against your changes.